src/types/EtherspotTransactionKit.ts

```diff
@@ -8,6 +8,14 @@ export interface ITransaction {
   data?: string;
 }
 
+export interface IProviderWalletTransaction {
+  id?: string;
+  to: string;
+  value?: BigNumberish;
```

Comment:
Do you think at least value should be a requirement also? I'm just thinking about the minimum required to send a transaction - unless we assume 0 if no value exists?

Reply:
no `value` or no `data` means `undefined` which can also go as `0` for value (or `0x0` for `data`), I can make it mandatory, but not sure it's needed, lmk

Reply:
Nah it's all good, as long as the developer can't make a mistake here then it's all good 👍